caching.md

```diff
@@ -0,0 +1,79 @@
+# Caching in Phoenix applications
+
+## Why?
+
+Despite the
+[speed of Phoenix applications](https://github.com/dwyl/learn-phoenix-framework#why),
+database queries are a bottleneck which can slow down your app. Oftentimes you
+may be loading the same or similar data anytime someone accesses a page. Caching
+previously loaded data can help speed up your app and avoid the query bottleneck!
+
+## What
```

Comment:
For consistency - would you be able to make this 'What?' with a '?' because the 'Why?' has one... the same goes for 'How', please can you add a '?'. Or remove them all, either way, just so long as it is consistent 😊